File: Objective-C/PassObject/ViewController.m
//
//  ViewController.m
//  PassObject
//
//  Created by Kittitat Rodphotong on 11/4/2558 BE.
//  Copyright © 2558 kittitat. All rights reserved.
//

#import "ViewController.h"
#import "ViewController2.h"

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
    
}

- (void) prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    if ([segue.identifier  isEqual: @"Pass"]) {
        ViewController2 *controller = segue.destinationViewController;
        controller.str = @"Hello";
        
    }
    
    
}

/*- (IBAction)button {
    if (self.loginIsValid) {
        [self performSegueWithIdentifier:@"Pass" sender:self];
        
    } else {
        
        NSLog(@"Login Fail");
    }

    
}*/

@end

File: Swift/SendVariable/ViewController.swift
//
//  ViewController.swift
//  SendVariable
//
//  Created by Kittitat Rodphotong on 11/17/2558 BE.
//  Copyright © 2558 FameSprinter. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        if segue.identifier == "Pass" {
            let controller = segue.destinationViewController as! ViewController2
            controller.str = "Hello"
            
        }
    }
}

